File: python/01-iniciante/problem1018.py
# Inputs

value = int(input())
notes = [100, 50, 20, 10, 5, 2, 1]

# Calculations and Ouputs

for i in notes:
    number = int(value/i)
    value = int(value % i)
    print(str(number) + " nota(s) de R$ %.2f" % i)

File: go/01-iniciante/problem1018.go
package main

import "fmt"

func main() {
    var value int
    notes := [7]int {100, 50, 20, 10, 5, 2, 1}

    fmt.Scanf("%d", &value)

    for _, i := range notes {
        number := value/i
        value = value % i

        note := fmt.Sprintf("%.2f", float64(i))
        fmt.Println(number, " nota(s) de R$", note)
    }
}
